src/scripts/main.js

```diff
@@ -358,3 +358,31 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+let table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in your HTML. If it doesn't, `document.querySelector('.dashboard')` will return `null`, and attempting to append rows to `null` will result in an error.